app/(egress_app)/LiveKitTemplate/page.tsx

```diff
@@ -38,12 +39,42 @@ const EgressTemplate = () => {
   }
 
   useEffect(() => {
-    const newRoom = new Room({ adaptiveStream: true });
-    EgressHelper.setRoom(newRoom, { autoEnd: true });
 
-    EgressHelper.onLayoutChanged((layout) => {
-        // Implement layout change logic if necessary
+    function handleTrackSubscribed(
+      track: RemoteTrack,
+      publication: RemoteTrackPublication,
+      participant: RemoteParticipant,
+    ) {
+      if (track.kind === Track.Kind.Audio || track.kind === Track.Kind.Video) {
+        track.setMuted(false)
```

Comment:
is this needed?

Reply:
nope, just a sanity check because things weren't working